cmd/backup/script.go

```diff
@@ -89,6 +90,21 @@ func newScript() (*script, error) {
 	}
 
 	s.file = path.Join("/tmp", s.c.BackupFilename)
+
+	extMap := map[string]string{
+		"Extension": fmt.Sprintf("tar.%s", s.c.BackupCompression),
+	}
+
+	tmplFileName, tErr := template.New("extension").Parse(s.file)
+	if tErr != nil {
+		return nil, fmt.Errorf("newScript: unable to parse backup file extension template: %w", tErr)
+	}
+	var bf bytes.Buffer
+	if tErr := tmplFileName.Execute(&bf, extMap); tErr != nil {
```

Comment:
It would also be possible to anonymously define the data being passed here
```suggestion
	if tErr := tmplFileName.Execute(&bf, map[string]string{
	    "Extension": "tar."+s.c.BackupCompression
	}); tErr != nil {
```